pkg/util/azureblob_util.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"strings"
+
+	"github.com/GoogleContainerTools/kaniko/pkg/constants"
+)
+
+//return storage accoutName, containerName and blobName from context string
+//format of context [accountName]/[containerName]/[PathToContextfile]
+
+func GetContainerAndBlob(context string) (accoutName string, containerName string, blobName string) {
```

Comment:
Consider using https://godoc.org/github.com/Azure/azure-storage-blob-go/azblob#NewBlobURLParts to parse the URL instead of doing this yourself.

Reply:
will do